projects/batfish/src/org/batfish/grammar/cisco/CiscoLexer.g4

```diff
@@ -2391,6 +2412,11 @@ GROUP_URL
    'group-url'
 ;
 
+GSHUT
```

Comment:
From your earlier email, this looks like the name of a variable from your configs, rather than a keyword